### ISSUE TITLE
Missing dependency on [xComputer]DomainJoin in Small Farm example

### ISSUE THREAD

**Issue by jsteenbeek** [PowerShell/SharePointDsc#287]
First of, thanks for the great work you guys are doing!

**Details of the scenario you try and problem that is occurring:**
I've been playing around with the Small Farm example:
`xSharePoint/Modules/xSharePoint/Examples/Small Farm/SharePoint.ps1`

I'm getting issues on the dependency "[xComputer]DomainJoin"

I've found I had to add the xComputerManagement resource;
`Import-DscResource -ModuleName xComputerManagement`

And implement DomainJoin in the configuration section as follows:
```
		xComputer DomainJoin {
			Name = $Node.NodeName 
			DomainName = $ConfigurationData.NonNodeData.DomainDetails.DomainName 
			Credential = $domainAdminCredential
		}
```

This resolved the issue for me. Either I didn't get something about how to set everything up, or there was a missing dependency. The first option seems likely.

**The DSC configuration that is using the resource:**

**Version of the Operating System and PowerShell the DSC Target Node is running:**
Server 2012 R2 Data Center, PS5, SP2016

**Version of the DSC module you're using:**
Current

**Comment by BrianFarnhill** [comment]
@jsteenbeek Thanks for reporting - we meant to take this out of the examples (so we focused on providing examples of just our resource and not the dependencies). I'll add this to the bug list and will get the examples cleaned up.

**Comment by BrianFarnhill** [comment]
This is resolved in our 1.0 release branch (attached to #304). We will merge it back to dev after it's finalised for release and then close this issue off

**Comment by BrianFarnhill** [comment]
Closing this issue as it has been resolved in #304 - thanks guys
